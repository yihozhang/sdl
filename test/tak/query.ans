0	0
1	1
2	4
3	14
4	53
5	223
6	1034
7	5221
8	28437
9	165859
10	1029803
11	6772850
12	46983238
13	342509396
14	-1679360619
15	-609391655
16	1647942757
17	167858933
18	-1991683488
19	772263370
20	1858016859
21	-681149672
22	1073498155
23	1709510200
24	-1083053424
25	510587836
26	-1656931639
27	-843894694
28	-1921372445
29	663888570
