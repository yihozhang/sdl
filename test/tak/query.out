29	663888570
28	-1921372445
27	-843894694
26	-1656931639
25	510587836
24	-1083053424
23	1709510200
22	1073498155
21	-681149672
20	1858016859
19	772263370
18	-1991683488
17	167858933
16	1647942757
15	-609391655
14	-1679360619
13	342509396
12	46983238
11	6772850
10	1029803
9	165859
8	28437
7	5221
6	1034
5	223
4	53
3	14
2	4
1	1
0	0
